src/javascript/app/base/redirect_popup.js

```diff
@@ -0,0 +1,55 @@
+const BinarySocket     = require('./socket');
+const isEuCountry      = require('../common/country_base').isEuCountry;
+const Client           = require('../base/client');
+const Login            = require('../../_common/base/login');
+
+const RedirectPopup = (() => {
+    const onLoad = () => {
+        BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
+            const has_dont_show = localStorage.getItem('redirect_popup');
+            const $popup        = $('#redirect_popup_container');
```

Comment:
since this is a new file, please use vanilla JS if possible for better performance. we only maintain jquery on old binary-static files or if there's no easier way